working-groups/user-interface/About.md

```diff
@@ -0,0 +1,44 @@
+# User-interface (UI) working group
+Hello and welcome. This page talks about the UI working group. If you have any queries about the UI working group after going through this page please feel free to reach out to us on [slack.](https://join.slack.com/t/mojaglobal/shared_invite/zt-o6ta1ug0-rVLjAo460~d7JbZ~HpFFtw)
+
+## What’s a working group?
+A working group is a group of people working together to achieve specified goals. The groups are domain-specific and focus on discussion or activity around a specific subject area.
+
+## About the UI working group
+<a href="#incubating"><img src="https://img.shields.io/static/v1?label=Status &message=Active &color=green" /></a>
+
+The UI working group is responsible for developing the visual components, user interactions and user experience of the UI-based projects under moja global.
+
+| Meeting details            |                                                                                                    |
+| -------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| Leads                      | [@Shloka Gupta](https://github.com/chicken-biryani?tab=followers) [@Soham Shah](https://github.com/sohamsshah) |
+| Days                       | Every Saturday (19th Feb onwards)                                                                                                 |
+| Time                       | 10:30 (GMT) / 16:00  (IST) / 5:30 (EST)                                                  |
+| On-going project           | Develop TSC working group notice board.                                                                        |
+| Agenda                     | You can have a look at the upcoming agenda [here.](agenda.md)                                                  |
+| Meeting Notes              | Have a look at the Meeting notes over [here.](MeetingNotes.md)                                                 |
+| Contact                    | [Slack.](https://join.slack.com/t/mojaglobal/shared_invite/zt-o6ta1ug0-rVLjAo460~d7JbZ~HpFFtw)                 |
+| Relevant channels on Slack | #community-website, #general, #reporting-tool, #user-interface                                                 |
+
+## What is the goal of this working group?
+
+We are trying to follow a “sprint-like” structure where on each “sprint” cycle we start with planning and then work towards the goals defined on that plan. You can know about the ongoing and upcoming work by attending our online meetings OR by looking at the agenda OR by going through the list of issues under the relevant repositories OR dropping a message on Slack.
```

Comment:
If you are following a sprint, how are issues managed? GitHub Issues, Trello or Jira? Mention it here.